akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -520,17 +450,6 @@ unreachable cluster node has been downed and removed.
 The easiest way to run **Worker Dial-in Example** example yourself is to try the
 @scala[@extref[Akka Cluster Sample with Scala](samples:akka-samples-cluster-scala)]@java[@extref[Akka Cluster Sample with Java](samples:akka-samples-cluster-java)].
 It contains instructions on how to run the **Worker Dial-in Example** sample.
- 
-## Node Roles
```

Comment:
this was moved to common/cluster-usage.md but not included?

Reply:
This is now in the main /typed/cluster.md, right before the FD

Reply:
this is lost from this page now?
I don't think we can expect them to find it in typed/cluster.md

Reply:
Fixing that ^